tests/.pest/snapshots/Countries/TunisiaTest/it_can_calculate_tunisia_holidays_with_data_set___2024____2024_.snap

```diff
@@ -12,31 +12,19 @@
         "date": "2024-04-09"
     },
     {
-        "name": "Eid al-Fitr 1. Day",
-        "date": "2024-04-10"
-    },
-    {
-        "name": "Eid al-Fitr 2. Day",
-        "date": "2024-04-11"
+        "name": "Eid al-Fitr",
```

Comment:
Have you verified the updated snapshots? Are they correct?

Reply:
i fixed it, it's translation problem